src/hdmf/common/resources.py

```diff
@@ -694,15 +754,27 @@ def to_norm_tsv(self, **kwargs):
         Write the tables in ExternalResources to individual tsv files.
         """
         folder_path = kwargs['path']
-        for child in self.children:
-            df = child.to_dataframe()
-            df.to_csv(folder_path+'/'+child.name+'.tsv', sep='\t', index=False)
+        files = [folder_path+'/'+child.name+'.tsv' for child in self.children]
+
+        for i in range(len(self.children)):
+            df = self.children[i].to_dataframe()
+            df.to_csv(files[i], sep='\t', index=False)
+
+        with zipfile.ZipFile('er.zip', 'w') as zipF:
+          for file in files:
+              zipF.write(file)
+
+        # remove tsv files
+        for file in files:
+            remove_test_file('./'+file)
```

Comment:
```suggestion
            if os.path.exists(path):
                os.remove('./'+file)
```

The `remove_test_file` function is a utility that is intended for use with test files. It has functionality that is slightly different than the above and it may be modified for the purpose of removing test files as the name suggests. The usage here in `to_norm_tsv` is different and so we should not use `remove_test_file`.